src/main/java/org/alliancegenome/curation_api/model/entities/AGMDiseaseAnnotation.java

```diff
@@ -23,7 +23,9 @@ public class AGMDiseaseAnnotation extends DiseaseAnnotation {
 
     @IndexedEmbedded(includeDepth = 1)
     @IndexingDependency(reindexOnUpdate = ReindexOnUpdate.SHALLOW)
-    @ManyToOne
+    @ManyToOne(cascade = {})
```

Comment:
Does the cascade in the @ManyToOne not work? I would think that should work or you might need to cascade = ALL?

Reply:
No, the `@ManyToOne` cannot have the cascade set to all, as that results in the deletion of the agm when a related agmDiseaseAnnotation gets deleted, which is not what we want (we only want it the other way around, deleting an agm should also delete the related agmDiseaseAnnotations). I've tested the code in this PR and this works as intended.

Reply:
I guess I would be in favor then of removing the `@ManyToOne(cascade = {})` -> `@ManyToOne` also you can define the relationship on the other side and put the Cascade.DELETE there. 

More generally, mixing `javax.persistence` annotations with `org.hibernate.annotations` is not a good idea.